cmd/etcd.go

```diff
@@ -14,20 +14,46 @@ import (
 	"github.com/urfave/cli"
 )
 
+const s3Endpoint = "s3.amazonaws.com"
+
 func EtcdCommand() cli.Command {
 	snapshotFlags := []cli.Flag{
 		cli.StringFlag{
 			Name:  "name",
-			Usage: "Specify Snapshot name",
+			Usage: "Specify snapshot name",
 		},
 		cli.StringFlag{
 			Name:   "config",
 			Usage:  "Specify an alternate cluster YAML file",
 			Value:  pki.ClusterConfig,
 			EnvVar: "RKE_CONFIG",
 		},
+		cli.BoolFlag{
+			Name:  "s3",
+			Usage: "Enabled backup to s3, set true or false",
+		},
+		cli.StringFlag{
+			Name:  "endpoint",
+			Usage: "Specify s3 endpoint url",
+			Value: s3Endpoint,
+		},
+		cli.StringFlag{
+			Name:  "accessKey",
```

Comment:
I am not sure how the cli library will handle camelCase for flags, maybe we should go with a dash `access-key` ?